im.kaidan.kaidan.json

```diff
@@ -51,15 +50,20 @@
             "name": "kaidan",
             "buildsystem": "cmake-ninja",
             "config-opts": ["-DI18N=1", "-DQUICK_COMPILER=0"],
-            "cleanup": [
-                "/share/icons/hicolor/128x128/apps/kaidan.png"
-            ],
```

Comment:
I think this is the only part that was wrong. Can't see the original build logs as they are too old.